docs/cave/bundles-and-procedures.md

```diff
@@ -1,48 +1,48 @@
 
 
-##  Bundles
+##  Displays
 
 ### **File > Load Display**
 
-Load a previously-saved bundle from within the AWIPS system.  The Open Bundle dialog allows you to select your own saved bundles as well as those saved by other users.
+Load a previously-saved display from within the AWIPS system.  The pop-up dialog allows you to select your own saved displays as well as those saved by other users.
 
-Each selected bundle will load its contents to new tabs which are named after the bundle file name (e.g. NAM_ThetaE).
+Each selected display will load its contents to new tabs which are named after the display file name (e.g. GFS_TempWind).
 
-![image](../images/KTZZHoCV5S.gif)
+![image](../images/load-display.gif)
 
-Most saved bundles will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
+Most saved displays will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
 
 ---
 
-### **File > Save Display**
-
-Save a product display within the AWIPS system, synching the bundle between CAVE and the EDEX server.
+### **Load Display from Local Disk**
 
----
+To load a previously-saved display from a path within the file directory of the workstation, select **File > Load Dislay** and then select the **File** button on the right to browse your local directories.
 
-### **File > Manage Bundles**
+![image](../images/load-local-display.gif)
 
-Select and remove a saved bundle under File > Manage Bundles, this will open the Delete Bundle dialog.  Select the file name and click **OK** and then confirm deletion to remove the saved file permanently.
+---
 
-![image](../images/fileMenuManageBundles.png)
+### **File > Save Display**
 
-![image](../images/delete_bundle2.png)
+Save a product display within the AWIPS system. This syncs the display file between CAVE and the EDEX server.
 
 ---
 
-### Load Bundle from Local Disk
-
-To load a previously-saved display from a path within the file directory of the workstation, select **File > Load Dislay** and then select the **File** button on the right to browse your local directories.
+### **File > Save Display Locally**
 
-![image](../images/XB6vQf78pl.gif)
+To save a product display to a path within the file directory of the workstation, select **File > Save Display Locally** and then select the **File** button on the right.
 
 ---
 
-### Save Bundle to Local Disk
+### **File > Manage Bundles**
+
+Select and remove a saved display under File > Manage Bundles, this will open a pop-up dialog.  Select the file name and click **OK** and then confirm deletion to remove the saved file permanently.
```

Comment:
the "File > Manage Bundles" should be bolded to be consistent with other uses of the file menu on this page?